@xen-orchestra/xapi/src/vbd.js

```diff
@@ -66,4 +67,23 @@ module.exports = class Vbd {
       await this.callAsync('VBD.plug', vbdRef)
     }
   }
+
+  async disconnect(ref) {
```

Comment:
IMHO, we should name this `unplug`, let's keep as close as XAPI as possible :slightly_smiling_face: